src/main/steps/c100-rebuild/safety-concerns/abduction/previousabductions/template.njk

```diff
@@ -0,0 +1,18 @@
+{% extends "common/template.njk" %}
+
+
+{% block page_content %}
+<span class="govuk-caption-xl">{{caption}}</span>
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
+<p class="govuk-body-m govuk-!-margin-bottom-1">{{line1}}</p>
+{% endblock %}
+{% block page_safetyConcerns %}
+<p class="govuk-body-m">
```

Comment:
@niranjan-301 Please remove these lines & include common template.njk file in abduction folder.

Reply:
done